openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MixedLoadTest.java

```diff
@@ -23,14 +23,31 @@
 import net.adoptopenjdk.stf.processes.definitions.LoadTestProcessDefinition;
 import net.adoptopenjdk.stf.runner.modes.HelpTextGenerator;
 import net.adoptopenjdk.loadTest.InventoryData;
+import net.adoptopenjdk.stf.environment.StfTestArguments; 
 
 public class MixedLoadTest implements StfPluginInterface {
+	
+	String timelimitInMins;
+	int testCountMultiplier;
+	boolean isTimeBasedLoadTest = true; 
+	
 	public void help(HelpTextGenerator help) throws StfException {
 		help.outputSection("MixedLoadTest runs workloads of math,mauve,nio,lang,concurrent unit tests");
 		help.outputText("");
 	}
 
 	public void pluginInit(StfCoreExtension stf) throws StfException {
+		// When used as a time based test, this STF test plugin accepts the following two parameters:
+		//   (1) timelimitInMins     : Time duration (e.g. 5s, 5m, 5h) for which the load should be run. 
+		//   (2) testCountMultiplier : Optional multiplier value to increase the workload externally.
+		StfTestArguments testArgs = stf.env().getTestProperties("timelimitInMins=[0m]","testCountMultiplier=[1]");
+		timelimitInMins = testArgs.get("timelimitInMins");
```

Comment:
Since the user can now spcify any time units (5ms, 5s, 5m or 5h) the variable should not have the 'InMins' ending.

Reply:
Makes sense. Updated the PR.